packages/web-components/src/components/footer/footer.scss

```diff
@@ -41,6 +41,21 @@
 :host(#{$dds-prefix}-footer),
 :host(#{$dds-prefix}-legal-nav) {
   display: block;
+  div.#{$prefix}--adjunct-links__container {
```

Comment:
Is the `div` selector needed here?

Reply:
No, not really, @kennylam. I've removed it!

Reply:
```suggestion
.#{$prefix}--adjunct-links__container {
```